source/common/config/utility.h

```diff
@@ -336,6 +336,18 @@ class Utility {
     return Registry::FactoryRegistry<Factory>::getFactoryByType(getFactoryType(typed_config));
   }
 
+  template <class Factory>
+  static Factory& getAndCheckFactoryByType(const ProtobufWkt::Any& typed_config) {
```

Comment:
Where is this used vs. being able to use the other function with TypedExtension?

Reply:
No longer being used, will remove. Previously I was accepting Any in the config, now updated to TEC